core/src/main/resources/error/error-classes.json

```diff
@@ -14,6 +42,10 @@
     "message" : [ "Found duplicate keys '%s'" ],
     "sqlState" : "23000"
   },
+  "FAILED_EXECUTE_UDF" : {
+    "message" : [ "Failed to execute user defined function (%s: (%s) => %s)" ],
+    "sqlState" : "45000"
```

Comment:
The SQLSTATE '45000' is not valid. Because it is not included in [README](https://github.com/apache/spark/blob/master/core/src/main/resources/error/README.md)

Reply:
@karenfeng I delete it.